.gitignore

```diff
@@ -99,16 +99,21 @@ oprofile_data/
 CMakeSettings.json
 /ci-artifacts/
 /out/
-/pcsx2-winrt/resources/cheats_ws.zip
-/pcsx2-winrt/resources/cheats_ni.zip
+
+# WINRT/UWP Files
+/pcsx2-winrt/resources/patches.zip
 /bin
 /AppPackages/pcsx2-winrt
 /packages
 /pcsx2-winrt/resources
 /pcsx2-winrt/Generated Files/winrt
 /pcsx2-winrt/BundleArtifacts
-/pcsx2-winrt/readme.txt
+/pcsx2-winrt/README.md
```

Comment:
Visual studio sometimes creates a 'readme.txt' file, so this probably shouldn't be removed.

Reply:
Both files are included as a compromise.